controlplane/hack/deploy/ns.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ocm-controlplane 
```

Comment:
the namespace should be configured by user because we need to deploy multi controlplane in a single OCP.

Reply:
Yes. we do not need to create namespace.

Reply:
done